examples/audit/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  flipt:
+    build: ../../
+    command: ["./flipt", "--force-migrate"]
+    volumes:
+      - /tmp/flipt/audit.log:/var/opt/flipt/audit.log
+    ports:
+      - "8080:8080"
+    environment:
+      - FLIPT_LOG_LEVEL=debug
+      - FLIPT_AUDIT_SINKS_LOG_ENABLED=true
+      - FLIPT_AUDIT_SINKS_LOG_FILE=/var/opt/flipt/audit.log
```

Comment:
```suggestion
      - FLIPT_AUDIT_SINKS_LOG_FILE=/var/log/flipt/audit.log
```

sorry again

Reply:
Tip: adding another path will need a chown in these locations:

- https://github.com/flipt-io/flipt/blob/main/Dockerfile#L44
- https://github.com/flipt-io/flipt/blob/main/hack/build/internal/flipt.go#L195

Reply:
@markphelps @GeorgeMac Yeah I used the location `/var/log/audit.log` to avoid modifying the `Dockerfile`. If I should actually create that `flipt` directory under `/var/log`, then I am down to do that too.